patches/react-native-screens+3.29.0+002+fixLayoutIssues.patch

```diff
@@ -0,0 +1,214 @@
+diff --git a/node_modules/react-native-screens/android/src/fabric/java/com/swmansion/rnscreens/FabricEnabledViewGroup.kt b/node_modules/react-native-screens/android/src/fabric/java/com/swmansion/rnscreens/FabricEnabledViewGroup.kt
+index 2d946c4..ccda8f3 100644
+--- a/node_modules/react-native-screens/android/src/fabric/java/com/swmansion/rnscreens/FabricEnabledViewGroup.kt
++++ b/node_modules/react-native-screens/android/src/fabric/java/com/swmansion/rnscreens/FabricEnabledViewGroup.kt
+@@ -12,38 +12,36 @@ import kotlin.math.abs
+ abstract class FabricEnabledViewGroup constructor(context: ReactContext?) : ViewGroup(context), FabricViewStateManager.HasFabricViewStateManager {
+     private val mFabricViewStateManager: FabricViewStateManager = FabricViewStateManager()
+ 
+-    private var lastSetWidth = 0f
+-    private var lastSetHeight = 0f
++    private var lastHeaderHeight: Double = 0.0
```

Comment:
what about this patch and the other react-native-screens patch? Have link(s) to upstream PRs? Can we upgrade the dependency and get rid of these patches?

Reply:
This one is no longer necessary due to other changes we made in `react-native-screens` repo. We will have to revisit it though if we ever want `native-stack` in the App.

Reply:
I'm confused - didn't we recently start using native-stack in E/App?

Reply:
IIRC it has been reverted and I don't see it in the `package.json`.